### PR TITLE
orm 3.0 support

### PR DESCRIPTION
**Pull request by konradkozaczenko**

### CODE REVIEW

File: tests/Translation/TranslatorTest.php [hunk 1]
```diff
@@ -86,19 +86,13 @@ public function testLoadCurrentLocale(
     ): void {
         $translator = new Translator($this->translator, $this->requestStack, $this->defaultLocale);
 
-        $request = $this->createMock(Request::class);
-        $request->query = $this->createMock(ParameterBag::class);
+        $request = new Request(['locale' => $requestedLocale]);
```
Comment (konradkozaczenko) [inline tests/Translation/TranslatorTest.php:89]: fix 
```
1) Locastic\ApiPlatformTranslationBundle\Tests\Translation\TranslatorTest::testLoadCurrentLocale with data set #0 ('en', 'en')
TypeError: Cannot assign Mock_ParameterBag_6af88324 to property Symfony\Component\HttpFoundation\Request::$query of type Symfony\Component\HttpFoundation\InputBag
```

**Comment by paullla** [comment]
Hey @konradkozaczenko thank you for this PR. Some tests are failing, can you take a look at them and then we can merge?

**Comment by konradkozaczenko** [comment]
> Hey @konradkozaczenko thank you for this PR. Some tests are failing, can you take a look at them and then we can merge?

Hi, doctrine/orm 3.0 is not compatible with this php8.0. I've updated php requirement in composer.json but I guess that's all I can do :(

**Comment by paullla** [comment]
For this version we can remove php 8.0 from checks since it is no longer supported and add php 8.3: https://github.com/Locastic/ApiPlatformTranslationBundle/blob/master/.github/workflows/phpunit.yml#L14

**Comment by paullla** [comment]
@konradkozaczenko thank you, I'll merge this and handle Scrutinizer issue in the next pr :)

**Pull request merged**